### ISSUE TITLE
What is input for custom_trainer_ex? 

### ISSUE THREAD

**Issue by Greenarrow289** [davisking/dlib#2005]
Hello,
first thing I would like to apologize for my English. Hope you understand :)
I trained my own face detection, I get mmod file and do a face detection. That works fine. But now I want to do a program on binary classification, that just decide if face is veiled or not. I looked on custom_trainer_ex but I don't know what is input for this algorithm? What I would expect to be input is some xml file, like in training, where should be a dataset of cropped faces which are veiled...than do some training. 
Thank you very much

**Comment by davisking** [comment]
I am suddenly getting questions from multiple people about this veiled face detection task, which makes me think this is a homework problem in some class.  This kind of question should be directed to your teachers or to stackoverflow.

**Comment by Greenarrow289** [comment]
If you mean that topic on sourceforge, that is me too. I found it after I start topic here, because the "help" section looked better for this question, than issue on github. 
You are right that it's my work to school. I talked with my teacher about binary classification but he said, that the input would be a group of image with veiled faces and group of normal faces, with labels (+1 and -1) and than do training. In dlib there are many binary classification examples and inputs are never images. So I asked here about the format of input. If it's something common maybe I should ask somewhere else or learn more about it...
